### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by nancymic2**
First commit is a test only

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,120 +23,168 @@ Today, the attestation model does not seek to provide information to users withi
...
 
-We strive to bring enhanced transparency to the use of the Privacy Sandbox relevance and measurement APIs. Therefore, besides placing the file on a server, we will generate reports that share enrollment and attestation related information for each enrolled developer. These reports will be designed to help a wide variety of audiences, including end users, governments and KOFs, gain more understanding and context around who is using these APIs. 
+We strive to bring enhanced transparency to the use of the Privacy Sandbox relevance and measurement APIs. Therefore, besides placing the file on a server, we will generate reports that share enrollment and attestation related information for each enrolled developer. These reports will be designed to help a wide variety of interested parties gain more understanding and context around who is using these APIs. 
+
+## Notes
```
Comment (rowan-m) [inline README.md:187]: I don't think this needs a separate heading, as it generates a double separating line.

File: README.md [hunk 2]
```diff
@@ -25,120 +23,168 @@ Today, the attestation model does not seek to provide information to users withi
...
+
 <table>
   <tr>
-   <td><strong>Service</strong></td>
-   <td>A 'service' reflects both client side APIs in the Privacy Sandbox and any related servers that altogether, colloquially represent an API. For example, the Protected Audience API includes both client side APIs in a browser or Android but also a variety of servers such as the key-value server, which inclusively are considered the <em>service</em>.</td>
-  </tr>
-  <tr>
-   <td><strong>NotUsed</strong></td>
-   <td>The public documented privacy goal of the Privacy Sandbox is based on protecting the user from a particular privacy abuse. By having the site provide a negative attestation via <em>NotUsed</em>,  that it is not violating that principle, the attestation can offer more flexibility for how a site can use the data from the services in new and innovative ways while still staying true to the privacy protecting principle we have laid out.</td>
+   <td><strong>Service</strong>
```
Comment (rowan-m) [inline README.md:62]: Update to <code> formatting for the terms as they reference the code formatted attestation above.

File: README.md [hunk 3]
```diff
@@ -25,120 +23,168 @@ Today, the attestation model does not seek to provide information to users withi
...
 
 Additional attestations may be added to individual APIs as scenarios arise.
 
 ### Validation and transparency framework
 
-Developers who submit the enrollment form will be sent a file that contains the attestations for the APIs they are requesting to use. To complete enrollment, the developer must place the file in a site's public `.well-known` location. For example, if they enroll with `example.com`, they need to place the attestation file at `https://example.com/.well-known/privacy-sandbox-attestations.json`. 
+Developers who submit an enrollment form are then sent a file that contains the attestations for the APIs they requested to use. To complete enrollment, the developer places the file in a site's public `.well-known` location. For example, if they enroll [example.com](https://example.com), they need to place the attestation file at `https://foo.com/.well-known/privacy-sandbox-attestations`.
```
Comment (rowan-m) [inline README.md:93]: Use `fixed formatting` around example.com as it should not be an actual link.

File: README.md [hunk 4]
```diff
@@ -25,120 +23,168 @@ Today, the attestation model does not seek to provide information to users withi
...
+   </td>
+   <td>Attestations for the Attribution Reporting APIs. This entry is a list, which contains a list of <code>&lt;Attestation, Boolean></code> items that indicate the attestation strings the ad tech attests to. 
+   </td>
   </tr>
   <tr>
-   <td><code>topics_api</code></td>
-   <td>Attestations for the Topics API. This entry is a list, which contains a list of <code>&lt;Attestation, Boolean&gt;</code> the attestation strings the ad tech attests to. </td>
+   <td><code>topics_api</code>
+   </td>
+   <td>Attestations for the Topics API. This entry is a list, which contains a list of <code>&lt;Attestation, Boolean></code>code> items that indicate the attestation strings the ad tech attests to.
```
Comment (rowan-m) [inline README.md:160]: Typo: extra `code>`.

**Review by rowan-m — CHANGES_REQUESTED** [review]
A few tweaks, but LGTM after that.

File: README.md (hunk 2 above)
Comment (nancymic2) [reply README.md:62]: ah I just changed from code as the last iteration used strong. Changing back.

**Review by rowan-m — APPROVED** [review]
LGTM - please squash commits on merge and add a meaningful overview.

File: README.md [hunk 5]
```diff
@@ -2,18 +2,16 @@
...
 
 In addition to these technical privacy guardrails, we [announced our plans](https://developer.chrome.com/blog/announce-enrollment-privacy-sandbox/) to launch a new developer enrollment process which will verify companies before they can use the APIs, as an additional layer of protection for user privacy. As part of this process, we propose an attestation model, which requires developers to agree to restrictions around the usage of these services to prevent re-identification of users across sites. 
 
-[^1]: Even though the Potential Privacy Model for the Web document was written with the web in mind, it still describes the core privacy threat that the Privacy Sandbox as a whole is designed to protect, and we apply the principle to both web and Android.
```
[the remark this answers is not in the document]
Comment (nancymic2) [reply README.md]: It refuses to move. Even when placed on line 9 @rowan-m

**Comment by nancymic2** [comment]
Update  the Enrollment Attestation explainer to reflect new language.

File: README.md [hunk 6]
```diff
@@ -25,120 +25,163 @@ Today, the attestation model does not seek to provide information to users withi
...
 
 Additional attestations may be added to individual APIs as scenarios arise.
 
 ### Validation and transparency framework
 
-Developers who submit the enrollment form will be sent a file that contains the attestations for the APIs they are requesting to use. To complete enrollment, the developer must place the file in a site's public `.well-known` location. For example, if they enroll with `example.com`, they need to place the attestation file at `https://example.com/.well-known/privacy-sandbox-attestations.json`. 
+Developers who submit an enrollment form are then sent a file that contains the attestations for the APIs they requested to use. To complete enrollment, the developer places the file in a site's public `.well-known` location. For example, if they enroll `example.com`, they need to place the attestation file at `https://foo.com/.well-known/privacy-sandbox-attestations`.
```
Comment (dmdabbs) [inline README.md:95]: Note that as of a week ago, your "enrollment complete" notification to registrants specified the extension. If you are removing it you should notify existing successful registrants who it is reasonable to assume have followed those instructions. 

>28 September 2023
From: Privacy Sandbox Enrollment Support <privacy-sandbox-enrollment@google.com>
>
>Hello,
Thank you for your Privacy Sandbox enrollment. We have completed your enrollment: XXXXXX
If you have not already done so, be sure to upload the attestation file to your site in the .well-known directory ([https://foo.com/.well-known/privacy-sandbox-attestations.json](https://foo.com/.well-known/privacy-sandbox-attestations.json))
>
>Best,
Rob & Privacy Sandbox Enrollment Support

Comment (dmdabbs) [inline README.md:95]: Typo. foo.com <> example.com

Comment (nancymic2) [reply README.md:95]: oops

**Pull request merged**